src/backend/distributed/sql/citus--11.3-1--12.0-1.sql

```diff
@@ -1,3 +1,21 @@
 -- citus--11.3-1--12.0-1
 
 -- bump version to 12.0-1
+
+CREATE TABLE citus.pg_dist_tenant_schema (
+    schema_id oid NOT NULL,
```

Comment:
use regnamespace instead?

Reply:
no, that breaks pg_upgrade https://www.postgresql.org/docs/current/pgupgrade.html

Reply:
> are schema OIDs stable across pg_upgrade?

I don't think so.

Reply:
> no, that breaks pg_upgrade https://www.postgresql.org/docs/current/pgupgrade.html

Given that we anyway need to save & reload pg_dist_tenant_schema during pg-upgrades (because oids are not stable), would it still be bad to use regnamespace?

Reply:
I cannot see any Postgres catalog tables that has a regnamespace column, so it might be nicer to keep using Oid.